handbook/sales/README.md

```diff
@@ -110,6 +110,37 @@ To schedule an [ad hoc meeting](https://www.vocabulary.com/dictionary/ad%20hoc)
  - Do not assign the issue. The Director of Solutions Consulting will assign the issue.
  - Ensure that the product category is defined ("Endpoint ops", "Device management", or "Vulnerability management") in the description of the issue.
 
+### Hand off technical evaluations in advance of vacation time
+At Fleet, we value vacation time. Fleeties should be able to disconnect completely without having to check email or slack while away from work. In advance of vacation time (target one week in advance), refer to the following examples and review with each individual that will act as the temporary DRI for the technical evaluation while you are away. This can be documented as a google doc or can be added to the relevant tech eval epic issue in github.
```

Comment:
Make this less about vacation and more about the handoff. This can be used in any scenario where a handoff is necessary.

Reply:
copy that.